package.json

```diff
@@ -10,7 +10,8 @@
   "license": "MIT",
   "peerDependencies": {
     "react": ">=15",
-    "react-native": ">=0.48"
+    "react-native": ">=0.48",
```

Comment:
Will this work alongside old versions of RN?

Reply:
react-native-webview's peer dependency says `>=0.57 <0.59`:
https://github.com/react-native-community/react-native-webview/blob/master/package.json#L25

Can you update this in the PR? I'll merge it.

Reply:
Done